### DISCUSSION TITLE
Fork + maven central

### DISCUSSION THREAD

**Discussion by Bixilon** [kotlin-graphics/glm#26] [Show and tell]
I just want to let you know, that I have created a fork of the project and improved a ton of underlaying stuff (like giving proper package names, class names, ...).

It is also now published on maven central. https://gitlab.bixilon.de/bixilon/kotlin-glm

I don't expect you to use it or use the code, probably you don't want most of the stuff. Maybe you can check some stuff.

**Comment by elect86** [comment]
Hello Bixilon,

my main concern is that I'd like to avoid wasting energies on multiple projects

would you like to discuss the changes to see what you could contribute here?
